addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -4206,6 +4206,53 @@ QUnit.test('auto-focus composer on opening thread', async function (assert) {
     );
 });
 
+QUnit.test('mark channel as seen if last message is visible when switching channels even when the previous channel had a more recent last message than the current channel', async function (assert) {
```

Comment:
Would remove "even" in the test name. It's confusing when the test actual test specifically this scenario, nothing else.